### PR TITLE
Add initial dashboard plugin package - Closes #6218, #6219, #6221, #6222

### LINKED ISSUE #6218: Setup Dashboard plugin package
### Description

Create a lisk-framework-dashboard-plugin under framework-plugin
- Add scaffold plugin which can be used for further development
- Add dashboard plugin class with configuration
  - Application URL (optional, , default: ws://localhost:8080)

### Acceptance Criteria

- Build should be passing
- It should have empty plugin class
- The setup should be the same as faucet plugin
- Add test to ensure all defaults are resolved as defined above
- Add test to ensure specified value is overriding the default
- Add test to ensure all configs are stored in memory

### Additional Information
Related Issue #6216

### LINKED ISSUE #6219: Setup UI package directories and libraries for dashboard
### Description

Add basic directory and library setup on the UI folder for the dashboard plugin
```
/ui
  index.tsx <= initializer
  app.tsx <= application top
  app.scss
  /pages <= holds smart components (only main page)
  /components <= holds dumb components
    /category1
       /component1.tsx
       /component1.module.scss
  /stores
    /actions <= defines set of actions
    /reducers <= defines set of reducers
    index.ts <= exposes store to application
  /styles <= contains scss which defines global style and constants

```
- It should use `@reduxjs/toolkit` to initialize the store

### Acceptance Criteria
- When started, empty scaffolded page should be shown

### Additional Information
Related Issue #6216

### LINKED ISSUE #6221: Create SCSS constant
### Description

Create SCSS constants to be used in the components.
- colors 
- font sizes
- spacings

For example,
```
$fontXS: 10px;
$fontS: 12px;
```

### Acceptance Criteria

All existing colors, font sizes and spacings are defined.

### Additional Information
Related Issues #6216

### LINKED ISSUE #6222: Create react component for text
### Description

Create a text component which can be used in all the cases
- color
- size
- bold

### Acceptance Criteria

- It should use the scss constants
- It should be able to produce all types of the text

### Additional Information
Related Issues: #6216

### PR DESCRIPTION
**Pull request by shuse2**
### Description

- Add basic structure of the plugin
- Add Text component for sample
- Add Redux setup
- Add CSS constants
- Add empty main page

### What was the problem?

This PR resolves #6218, resolves #6219, resolves #6221, resolves #6222

### How was it solved?
- Copied most of the structure from the faucet plugin
- Add CSS constants according to the design
- Add `Text` component as a sample

### How was it tested?

- Run `npm run start:web` and see the design
- Check folder structure

### CODE REVIEW

File: framework-plugins/lisk-framework-dashboard-plugin/.prettierignore [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# project specific paths
+coverage/
+dist-node/
+dist-browser/
+dist/
+tmp/
+browsertest.build/
+.coverage
+.coverage-unit
+src/bootstrapping/templates/lisk-template-ts/templates/**/*
```
Comment (ishantiw) [inline framework-plugins/lisk-framework-dashboard-plugin/.prettierignore:38]: is it required here or is it the standard ignore list

File: framework-plugins/lisk-framework-dashboard-plugin/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+```sh
+$ npm install --save @liskhq/lisk-framework-dashboard-plugin
+```
+
+## Config Options
+
+```
+{
+	applicationURL?: string,
+  host?:string,
```
Comment (ishantiw) [inline framework-plugins/lisk-framework-dashboard-plugin/README.md:16]: formatting looks off

File: framework-plugins/lisk-framework-dashboard-plugin/public/index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (ishantiw) [inline framework-plugins/lisk-framework-dashboard-plugin/public/index.html:28]: Should change the title to something like `Lisk Dashboard Application`, I think we should do it for `lisk-framework-faucet-plugin` too

File: framework-plugins/lisk-framework-dashboard-plugin/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+			"src": "logo192.png",
+			"type": "image/png",
+			"sizes": "192x192"
+		},
+		{
+			"src": "logo512.png",
+			"type": "image/png",
+			"sizes": "512x512"
+		}
+	],
```
Comment (ishantiw) [inline framework-plugins/lisk-framework-dashboard-plugin/public/manifest.json:20]: These icons are React logos, is it fine to use them?

File: framework-plugins/lisk-framework-dashboard-plugin/scripts/clean.sh [hunk 5]
```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+#
+# Copyright © 2019 Lisk Foundation
```
Comment (ishantiw) [inline framework-plugins/lisk-framework-dashboard-plugin/scripts/clean.sh:4]:
```suggestion
# Copyright © 2021 Lisk Foundation
```

File: framework-plugins/lisk-framework-dashboard-plugin/src/plugin/defaults/config.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+			maximum: 65535,
+		},
+		host: {
+			type: 'string',
+			format: 'ip',
+		},
+	},
+	required: [],
+	default: {
+		applicationUrl: 'ws://localhost:8080',
```
Comment (ishantiw) [inline framework-plugins/lisk-framework-dashboard-plugin/src/plugin/defaults/config.ts:36]:
```suggestion
		applicationUrl: 'ws://localhost:8080/ws',
```

**Review by ishantiw — APPROVED** [review]

File: framework-plugins/lisk-framework-dashboard-plugin/src/plugin/dashboard_plugin.ts [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+	public get actions(): ActionsDefinition {
+		return {};
+	}
+
+	// eslint-disable-next-line @typescript-eslint/require-await, class-methods-use-this
+	public async load(_channel: BaseChannel): Promise<void> {
+		this._options = objects.mergeDeep(
+			{},
+			defaults.config.default,
+			this._options,
```
Comment (mitsuaki-u) [inline framework-plugins/lisk-framework-dashboard-plugin/src/plugin/dashboard_plugin.ts:74]:
```suggestion
			this.options,
```
In other plugins it merges `this.options` from the `BasePlugin` class

**Review by mitsuaki-u — APPROVED** [review]

**Pull request merged**